app/components/sign-up.jsx

```diff
@@ -384,6 +386,17 @@ const useStyles = createUseStyles(theme => ({
     marginBottom: '2rem',
     borderRadius: '0.5rem !important',
     boxSizing: 'border-box !important',
+    '&::placeholder': {
+    color: theme.colors.title, 
+    fontSize: '1rem', 
+    fontFamily: 'Inter',
+    },
+    '&[type="password"]::placeholder': {
+      color: '#0A0A0A',
```

Comment:
This color should be in theme.js, and referenced here.  Look above to see how theme.colors.... is used. 

But this color still too dark compared to what you see in figma.  Please figure out the matching color, and you might need to apply opacity.

Also, after I type in the input field, I see the text is smaller than the placeholder text.  I think the place holder text is the right size, and the input field text needs to be set to this size.  Please take care of that too.